kivymd/uix/textfield.py

```diff
@@ -426,6 +426,8 @@ def build(self):
 from kivymd.theming import ThemableBehavior
 from kivymd.uix.label import MDIcon
 
+import re
+
```

Comment:
Thanks for making these changes!
But could you move the modules of the standard library up.

Reply:
Sure.